packages/mediation-manager/README.md

```diff
@@ -0,0 +1,124 @@
+# Veramo: Mediation Manager Plugin
+
+A plugin for the Veramo agent for integration with the DIDComm plugin when implementing mediation to the [Cooordinate Mediation V3 Specification](https://didcomm.org/coordinate-mediation/3.0/).  
+
+## setup
+
+The Mediation Manager Plugin can be used via the `agent.yml` or programatically injected to the DIDComm plugin. 
+
+### example agent.yml configuration
+
+`agent.yml` example configuration for the `mediationManager` plugin below:
+
+```yml
+# Database
+dbConnection:
+  $require: typeorm#DataSource
+  $args:
+    - type: sqlite
+      database:
+        $ref: /constants/databaseFile
+      synchronize: false
+      migrationsRun: true
+      migrations:
+        $require: '@veramo/data-store?t=function#migrationConcat'
+        $args:
+          - $require: '@veramo/data-store?t=object#migrations'
+          - $require: '@veramo/kv-store?t=object#kvStoreMigrations'
```

Comment:
neat solution for combining migrations!

Reply:
Credit goes to @pauldesmondparker for this one!